CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+0.3.3
+----
+Add EKS module for AWS.
+Modified aws alb module to allow creation of only the alb.
+Added an eks_alb_controller module to use with the EKS cluster.
+Added local helm template example.
```

Comment:
Move these to RELEASENOTES
They will automatically added with the correct new version once the PR is merged and the actions publish the changes.